src/components/ButtonGroup/ButtonGroup.tsx

```diff
@@ -26,7 +26,7 @@ export const ButtonGroup = ({
 }: Props) => {
     const totalNumberOfButtons = React.Children.count(children);
 
-    if (totalNumberOfButtons === 1) {
+    if (totalNumberOfButtons === 1 && !Array.isArray(children)) {
```

Comment:
As i understand this part was added to avoid  this:
![image](https://user-images.githubusercontent.com/104063210/207273472-3726a7ba-c2e9-428b-b6aa-870c473c04d7.png)

Reply:
Now we have for one button class `first` and `last`.